tests/run-pass-noseed/intptrcast.rs

```diff
@@ -1,8 +1,13 @@
 // compile-flags: -Zmiri-seed=0000000000000000
 
+// This returns a miri pointer at type usize, if the argument is a proper pointer
+fn transmute_ptr_to_int<T>(x: *const T) -> usize {
+    unsafe { std::mem::transmute::<*const T, usize>(x) }
```

Comment:
```suggestion
    unsafe { std::mem::transmute(x) }
```
Type annotations are redundant here.